backend/app/services/mongo/explore.py

```diff
@@ -3038,9 +3039,9 @@ async def get_clustering_by_id(
         pipeline.append(
             {
                 "$project": {
-                    "_id": 0,
-                    "pca": 0,
-                    "tsne": 0,
+                    "_id": str(0),
+                    "pca": str(0),
+                    "tsne": str(0),
```

Comment:
Don't do that and instead change the type hint of pipeline
pipeline: List[Dict[str, object]]